medusa/post_processor.py

```diff
@@ -287,6 +288,16 @@ def list_associated_files(self, file_path, base_name_only=False, subtitles_only=
 
         return file_path_list
 
+    @staticmethod
+    def rar_basename(filepath, filelist):
+        videofile = os.path.basename(filepath)
+        rars = (x for x in filelist if rarfile.is_rarfile(x))
+
+        for rar in rars:
+            content = rarfile.RarFile(rar).namelist()
```

Comment:
We don't want to get the name of the video in the rar, but the name of the rar that holds the video we are processing. That's exactly what the code does.